### PR TITLE
test(github): fix extraction dates and correct PR count

### PR DESCRIPTION
**Pull request by davinov**
Fixes the tests that @luc-leonard noted were failing from November 18th.

### CODE REVIEW

File: tests/github/test_github.py [hunk 1]
```diff
@@ -384,8 +389,7 @@ def test_get_pages(
         )
     )
     assert mocked_api_call.call_count == 2
-    assert len(pr_rows) == 3
-    assert pr_rows[-1]['PR Name'] == 'chore(something): somethin'
```
Comment (davinov) [inline tests/github/test_github.py:388]: I'm not very sure why there was only 3 PRs displayed before. There is 3 sent in `extracted_prs_1` plus 1 in `extracted_prs_2`, so I think it's normal we change this to 4.

Comment (luc-leonard) [reply tests/github/test_github.py:388]: ah ah this left me wondering where this 3 was coming from, thanks for resolving this issue

File: tests/github/conftest.py [hunk 2]
```diff
@@ -490,3 +491,11 @@ def extracted_prs_4():
...
         }
     }
+
+
+@pytest.fixture
+def mock_extraction_start_date(mocker):
+    return mocker.patch(
+        'toucan_connectors.github.github_connector.extraction_start_date',
+        datetime.datetime(2020, 7, 1),
+    )
```
Comment (luc-leonard) [inline tests/github/conftest.py:501]: Nice solution

**Review by luc-leonard — APPROVED** [review]
LGTM! merci David!

**Pull request merged**